docs/supported_formats/mrc.rst

```diff
@@ -3,22 +3,56 @@
 MRC format
 ----------
 
-The ``.mrc`` format is widely used for tomographic data. Our implementation is based
-on `this specification
-<https://www2.mrc-lmb.cam.ac.uk/research/locally-developed-software/image-processing-software/>`_. We also
-partly support FEI's custom header. We do not provide writing features for this
-format, but, as it is an open format, we may implement this feature in the
-future on demand.
-
-.. Note ::
-    For ``.mrc`` files, the ``file_reader`` takes the ``mmap_mode`` keyword argument
+The ``mrc`` format is widely used for tomographic data. The implementation of this plugin
```

Comment:
We currently write the format extensions with a dot for most file formats, e.g. ``.bcf`` ...